l10n_it_ricevute_bancarie/models/account/account.py

```diff
@@ -194,19 +196,21 @@ def action_move_create(self):
             # ---- Apply Collection Fees on invoice only on first due date of the month
             # ---- Get Date of first due date
             move_line = self.env['account.move.line'].search([
-                ('partner_id', '=', invoice.partner_id.id)])
+                ('partner_id', '=', invoice.partner_id.id),
+                ('invoice_id.payment_term_id.riba', '=', True)
```

Comment:
Per evitare la ricerca su movimenti la cui scadenza è superata.
```suggestion
                ('invoice_id.payment_term_id.riba', '=', True),
                ('date_maturity', '>=', fields.Date.context_today(invoice))
```